backend/src/main/java/com/javahelp/backend/endpoint/HTTPTokenHandler.java

```diff
@@ -31,7 +31,7 @@ public abstract class HTTPTokenHandler extends HTTPHandler implements ITokenAuth
     private String tokenString, userId;
 
     @Override
-    public APIGatewayResponse getResponse(JsonObject body, HttpMethod method, Map<String, String[]> headers, Map<String, String[]> parameters) {
+    public APIGatewayResponse getResponse(JsonObject body, HttpMethod method, Map<String, String[]> headers, Map<String, String[]> parameters, Map<String, String> pathParameters) {
```

Comment:
Would this extra parameter be reflected in some other changes somewhere else?

Reply:
That should be everywhere